projects/uitpas/docs/faq.md

```diff
@@ -41,6 +41,6 @@ If the passholder doesn't end up buying the ticket, you should [cancel the ticke
 
 The calculation of the correct UiTPAS tariffs is dependent on many factors such as the availability of a social tariff for the passholder and settings within the UiTPAS region. These are checked in realtime by UiTPAS. That's why it's a bad idea to only use UiTPAS numbers without accessing the UiTPAS API.
 
-### My application is used by multiple UiTPAS organizers.  Can I use the same set of credentials?
+### My application is used by multiple UiTPAS organizers.  Can I use the same set of client access tokens?
```

Comment:
> Can I use the same set of client access tokens?

Misschien dit gewoon "credentials" laten? Of herschrijven naar `Can I use the same client access token for multiple organizers`?

Reply:
Ik wou even impliciet het onderscheid maken met user access tokens (die nog steeds bestaan en gedocumenteerd zijn) en waar de permissies heel anders georganiseerd worden. Je laatste suggestie is wel een goeie vereenvoudiging van de zin.